test/unit/ngsiv2/examples/subscriptionRequests/simpleSubscriptionRequest.json

```diff
@@ -0,0 +1,23 @@
+{
+  "subject": {
+    "entities": [
+      {
+        "id": "FirstMicroLight",
+        "type": "MicroLights"
+      }
+    ],
+    "condition": {
+      "attrs": [
+        "attr_name"
+      ]
+    }
+  },
+  "notification": {
+    "http": {
+      "url": "http://smartGondor.com/notify"
+    },
+    "attrs": [],
+    "attrsFormat": "normalized"
+  },
```

Comment:
Different to bidirectaionalSubscription.json, in this case the "expires" field is missing. However, looking to https://github.com/dcalvoalonso/iotagent-node-lib/pull/4/files#diff-c1822c96fd83111a79946a0cff296d6cR48

```
deviceRegistrationDuration: 'P1M',
```

I have the feeling that "expires" should be generated and captured by the mock using this .json (in particular with a value of 1 month in the future with regards the moment in which the test is run). 

Maybe I'm missing something... could you elaborate, please?

Reply:
This JSON payload is used to mock the request to create a subscription in the CB as part of the following test https://github.com/dcalvoalonso/iotagent-node-lib/blob/task/bidirectionalityPlugin/test/unit/ngsiv2/ngsiService/subscriptions-test.js#L80.
As you can see, the expected `expires` field is calculated dynamically because it depends on the real moment when the request is sent.

Reply:
Maybe a comment on L80 explaining could help. Something like:

> Note that expired field ins not included in the json used by this mock as it is a dynamic field. The following code performs such calculation and adds the field to the subscription payload of the mock.

or a similar wording.

Reply:
Fixed in 7ec0721596093dc03b2c2db6a471f3a5503d387c